engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -176,6 +176,13 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
             // the log.
             if (vrProvider.init()) {
                 playerCamera = new OpenVRStereoCamera(vrProvider);
+                /*
+                * The origin of OpenVR's coordinate system lies on the ground of the user. We have to move this origin
+                * such that the ground plane of the rendering system and the ground plane of the room the VR user is
+                * in match.
+                 */
+                //
+                vrProvider.getState().setGroundPlaneYOffset(-0.80f - context.get(Config.class).getPlayer().getEyeHeight());
```

Comment:
- no magic numbers: what's -0.80f?
- you might want to store the result of getPlayer() somewhere, so that you don't have to get it through the context every time this method is called.